tests/test_main.py

```diff
@@ -90,3 +90,27 @@ def test_print_full(tmpdir, capsys):
     assert 'environment:' in out
     assert 'PYTHONPATH:' in out
 
+
+def test_version(capsys):
+    """
+    Test --version flag.
+    """
+    from conda_devenv._version import version
+    assert devenv.main(['--version']) == 0
```

Comment:
There is any chance we will "compile to optimised byte code|"?

Reply:
Don't think so